src/Types/EnumType.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Mpociot\LaravelTestFactoryHelper\Types;
+
+use Doctrine\DBAL\Types\Type;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+
+/**
+ * My custom datatype.
+ */
+class EnumType extends Type
+{
+
+    const ENUM = 'enum'; // modify to match your type name
+
+    public function getSQLDeclaration(array $fieldDeclaration, AbstractPlatform $platform)
+    {
+        // this is wrong, but it also isn't important, since we're not modifying tables anyways. (this method must exist to match the abstract parent class, but we won't use it)
```

Comment:
Improve comment - what _should_ this be under normal circumstances. Also wrap at 80 characters.